daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/SAXInfosetInputter.scala

```diff
@@ -207,7 +207,8 @@ class SAXInfosetInputter(
       }
     } catch {
       case e: Exception => {
```

Comment:
Catch of Exception requires documentation. As part of figuring out how to get the change line covered by a test, you should add comments to this scala code as to why catching Exception is ok in this case. 

Does anyone (@stevedlawrence @olabusayoT ) know why this is ok? Normally catching Exception or Throwable is almost always an error. Please chime in.  

If this in fact should not be Exception, but something narrower, then let's fix that.